pkg/codegen/generator/cleanup/main.go

```diff
@@ -11,8 +11,13 @@ func main() {
 }
 
 func run() error {
-	if err := os.RemoveAll("../../clients/rancher/generated"); err != nil {
+	if err := os.RemoveAll("./pkg/generated"); err != nil {
 		return err
 	}
+
+	if err := os.RemoveAll("./clients/rancher/generated"); err != nil {
```

Comment:
shouldn't this be `("../../clients/rancher/generated")` ?

Reply:
Great question, the previous version wasn't cleaning up properly, and the path is not relative because of this: https://github.com/rancher/shepherd/blob/main/generator.go#L2